README.md

```diff
@@ -6,22 +6,19 @@
 [![Dependency Status][david-image]][david-url]
 [![devDependency Status][david-dev-image]][david-dev-url]
 
+> *This module is experimental, and should be used by early adopters. This module uses APIs there may be undocumented and may be subject to change without notice.*
 
-## Overview
-
-*This module is experimental, and should be used by early adopters. This module uses APIs there may be undocumented and may be subject to change without notice.*
-
-The Cloud Debugger is a feature of the Google Cloud Platform that lets you inspect the state of your application at any code location without stopping your application. For more information about the Cloud Debugger, see https://cloud.google.com/tools/cloud-debugger/. Here's an introductory video:
+This module provides Cloud Debug support for Node.js applications. [Google Cloud Debug](https://cloud.google.com/tools/cloud-debugger/) is a feature of [Google Cloud Platform](https://cloud.google.com/) that lets you debug your applications in production without stopping or pausing your application. Here's an introductory video:
 
 [![Cloud Debugger Intro](http://img.youtube.com/vi/tyHcK_kAOpw/0.jpg)](https://www.youtube.com/watch?v=tyHcK_kAOpw)
 
 ## Prerequisites
-* You are using Node.js 0.12+ for your application. (Node.js v5+ is recommended).
-* The source of your application is uploaded to your cloud project's [source repository](https://cloud.google.com/tools/cloud-repositories/docs/). The Debugger UI needs the source to be available in order to set breakpoints.
+* Your application will need to be using Node.js version 0.12 or greater. Node.js v5+ is recommended.
+* The source of your application is uploaded to a [cloud source repository](https://cloud.google.com/tools/cloud-repositories/docs/). The Debugger UI needs the source to be available in order to set breakpoints.
 
 ## Quick Start (Node.js v4.x+)
 ```shell
-# Install module, and save it as a dependency
+# Install with `npm` or add to your `package.json`.
```

Comment:
I see the way you are interpreting it, but assuming the reader is familiar with npm, it should be clear what this means.